### ISSUE TITLE
Monet palette not working for adw-gtk3

### ISSUE THREAD

**Issue by 0xMRTT** [AdwCustomizerTeam/AdwCustomizer#109]
Monet palette not working for adw-gtk3

_Originally posted by @daudix-UFO in https://github.com/AdwCustomizerTeam/AdwCustomizer/discussions/103#discussioncomment-3379573_

**Comment by 0xMRTT** [comment]
Think that's related to flatpak. Did you followed the setup for flatpak in read me?

**Comment by daudix-UFO** [comment]
Yes, it also don't work for native packages

**Comment by 0xMRTT** [comment]
Only when generating the theme ? Or every time even if you just customize yourself ?

**Comment by 0xMRTT** [comment]
What's colour mode do you use? What's colour mode did you selected during the apply.? Did you checked apply to adw-gtk3?

**Comment by daudix-UFO** [comment]
No, only monet don't work, but for example pretty purple works everywhere

**Comment by 0xMRTT** [comment]
Strange

**Comment by daudix-UFO** [comment]
Tried light/dark/auto modes

**Comment by daudix-UFO** [comment]
Is it works on your system?

**Comment by 0xMRTT** [comment]
Yes...

**Comment by 0xMRTT** [comment]
Maybe changed something and now it isn't working

**Comment by daudix-UFO** [comment]
OK, will try to reinstall adw-gtk3 and re-setup everything

**Comment by 0xMRTT** [comment]
Really need to add troubleshooting in about dialog

**Comment by daudix-UFO** [comment]
And clean all AdwCustomizer data

**Comment by 0xMRTT** [comment]
> OK, will try to reinstall adw-gtk3 and re-setup everything

I think that's not necessary... I can't check the code yet. I'll try to resolve latter. Sorry

**Comment by daudix-UFO** [comment]
Don't worry

**Comment by 0xMRTT** [comment]
So, is it working ?

**Comment by daudix-UFO** [comment]
I noticed on thing, after UI update there is no selector to apply it to libadwaita + adw-gtk3

**Comment by daudix-UFO** [comment]
No

**Comment by daudix-UFO** [comment]
Also, turns out that presets stopped working too

**Comment by daudix-UFO** [comment]
When you press apply, there is no switch to applying for adw-gtk3
![image](https://user-images.githubusercontent.com/77155297/184520831-4f7dcb96-d392-4c0a-8de1-0d539bf0e214.png)

**Comment by daudix-UFO** [comment]
Also, I tested some old version of AdwCustomizer, and it works
